storybook/webpack.config.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+const MiniCssExtractPlugin = require( 'mini-css-extract-plugin' );
+
+/**
+ * Internal dependencies
+ */
+const { getAlias, getMainConfig } = require( '../bin/webpack-helpers.js' );
+
+module.exports = ( { config: storybookConfig } ) => {
+	const wooBlocksConfig = getMainConfig( { alias: getAlias() } );
+
+	storybookConfig.resolve.alias = {
+		...storybookConfig.resolve.alias,
+		...wooBlocksConfig.resolve.alias,
+	};
+
+	storybookConfig.module.rules.push( ...wooBlocksConfig.module.rules );
+
+	storybookConfig.plugins.push(
+		new MiniCssExtractPlugin( {
+			filename: `[name].css`,
+		} )
+	);
+
+	return storybookConfig;
+};
```

Comment:
Good work on figuring out this config!  It still feels like there may be too much being included from the main config.  I did play around with it myself but didn't get too far so we can just leave this as is for now and might be something that could be experimented with in follow-ups.

Reply:
We can merge in specific `resolve.alias` items and `module.rules` (instead of wholesale) if needed later. I figure the more consistent our storybook environment is with normal build the better :)